modules/test/playwright/tests/style-book-web/previewStyleBookOnDisplayPageTemplates.spec.ts

```diff
@@ -0,0 +1,221 @@
+	});
+
+	await test.step('Create and publish a new page template for blogs entry', async () => {
+		await displayPageTemplatesPage.goto(site.friendlyUrlPath);
+
+		await displayPageTemplatesPage.createTemplate({
+			contentType: 'Blogs Entry',
+			name: displayPageTemplateNameBlogsEntry,
+			publish: false,
+		});
+
+		await placeHeadingFragmentAndSelectText();
+
+		await page.getByLabel('Source').selectOption('structure');
+
+		await page.getByLabel('Field').selectOption('BlogsEntry_content');
+
+		await displayPageTemplatesPage.publishTemplate();
+	});
+
+	await test.step('Create style book and view the display page template is selected in the preview type selector', async () => {
+		await styleBooksPage.goto(site.friendlyUrlPath);
+
+		await styleBooksPage.createStyleBook(styleBookName);
+		await styleBooksPage.publishStyleBook();
+
+		await styleBooksPage.editStyleBook(styleBookName);
+
+		await expect(
+			page.getByRole('button', {name: 'Display Page Templates'})
+		).toBeVisible();
+	});
+
+	await test.step('View the items shown in dropdown menu of preview item selector then change the preview item for each display page template', async () => {
+		const selectOptionAndUpdateColor = async (
+			selectButton: string,
+			selectMenuItem: string,
+			colorHEX: string,
+			colorRGB: string
+		) => {
+			await page.getByRole('button', {name: selectButton}).click();
+			await page.getByRole('menuitem', {name: selectMenuItem}).click();
+
+			await styleBooksPage.updateTokenInputColor('Body Color', colorHEX);
+
+			expect(page.getByText('Saved')).toBeVisible();
+
+			const heading = page
+				.frameLocator('iframe')
+				.getByRole('heading', {name: 'Heading Example'});
+
+			const fragmentColor = await heading.evaluate((element) => {
+				const computedStyle = window.getComputedStyle(element);
+
+				return computedStyle.color;
+			});
+
+			expect(fragmentColor).toBe(colorRGB);
+		};
+
+		await page.getByRole('button', {name: 'Color System'}).click();
+		await page.getByRole('menuitem', {name: 'General'}).click();
+
+		await selectOptionAndUpdateColor(
+			displayPageTemplateNameBlogsEntry,
+			displayPageTemplateNameBlogsEntry,
+			'#995511',
+			'rgb(153, 85, 17)'
+		);
+
+		await selectOptionAndUpdateColor(
+			displayPageTemplateNameBlogsEntry,
+			displayPageTemplateNameWebContent,
+			'#666666',
+			'rgb(102, 102, 102)'
+		);
+	});
+});
+
+test('View more button to select a display page template from select modal and change preview', async ({
+	displayPageTemplatesPage,
+	page,
+	site,
+	styleBooksPage,
+}) => {
+	const displayPageTemplateNameBlogsEntry = 'Blogs ' + getRandomString();
+	const displayPageTemplateNameCategory = 'Category ' + getRandomString();
+	const displayPageTemplateNameDocuments = 'Documents ' + getRandomString();
+	const displayPageTemplateNameWebContent1 = 'Web-1 ' + getRandomString();
+	const displayPageTemplateNameWebContent2 = 'Web-2 ' + getRandomString();
+	const styleBookName = getRandomString();
```

Comment:
I think having the prefix "blogs" and "WebContent" are unnecessary, you are generating a random string anyway. 

You are not adding a prefix for the `styleBookName` variable